rosdep/base.yaml

```diff
@@ -1509,6 +1511,10 @@ libgazebo5-dev:
   fedora: [gazebo-devel]
   gentoo: [sci-electronics/gazebo]
   ubuntu: [libgazebo5-dev]
+libgazebo7-dev:
+  arch: [gazebo]
+  gentoo: [sci-electronics/gazebo]
```

Comment:
Is there a way to ensure it's gazebo7 though? If you depend on `libgazebo7-dev`, then rosdep (and by translation the package manager for the OS) better give you version 7. It would be fine to have them for the `gazebo` key.